src/braket/circuits/gates.py

```diff
@@ -623,21 +624,31 @@ def rx(
         angle: Union[FreeParameterExpression, float],
         *,
         control: Optional[QubitSetInput] = None,
+        control_state: Optional[ControlState] = None,
     ) -> Iterable[Instruction]:
         """Registers this function into the circuit class.
 
         Args:
             target (QubitSetInput): Target qubit(s).
             angle (Union[FreeParameterExpression, float]): Angle in radians.
             control (QubitSetInput): Control qubit(s).
+            control_state (Optional[ControlState]): Quantum state on which to control the operation.
+                Must be a binary sequence of same length as number of qubits in `control`. Will be
+                ignored if `control` is not present. May be represented as a string, list, or int.
+                For example "0101", [0, 1, 0, 1], 5 all represent controlling on qubits 0 and 2
+                being in the |0⟩ state and qubits 1 and 3 being in the |1⟩ state.
+                Default "1" * len(control).
 
         Returns:
             Iterable[Instruction]: Rx instruction.
 
         Examples:
             >>> circ = Circuit().rx(0, 0.15)
         """
-        return [Instruction(Rx(angle), target=qubit, control=control) for qubit in QubitSet(target)]
+        return [
+            Instruction(Rx(angle), target=qubit, control=control, control_state=control_state)
+            for qubit in QubitSet(target)
+        ]
```

Comment:
I will need to add `control_state` to the rest of the gates as well

Reply:
Behavior is unobvious when applied to by-default controlled gates, so my plan as of now is to not allow supplying control_state for e.g. cnot. Open to alternatives.